openmdao/visualization/meta_model_viewer/tests/test_unstruct.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+import numpy as np
+import openmdao.api as om
+from openmdao.visualization.meta_model_viewer.meta_model_visualization import UnstructuredMetaModelVisualization
```

Comment:
I think we can add UnstructuredMetaModelVisualization to openmdao.api.

Also, why not just call it MetaModelVisualization.  We can detect which kind of metamodel it is in init, and run it accordingly.